apps/web/src/routes/index.tsx

```diff
@@ -334,12 +333,16 @@ function Home() {
       fetchConversionRate();
     }, []);
 
+    const upcomingEvent = events?.find(
```

Comment:
Not good -> not getting the immediate next event or currently happening event -> should be sorted by date before
(or better -> get only the concerned event from backend instead of doing a request fetching all the events each time someone goes on homepage)